modules/xmpp/ChatRoom.js

```diff
@@ -418,7 +418,7 @@ export default class ChatRoom extends Listenable {
                 && this.options.hiddenDomain
                     === jid.substring(jid.indexOf('@') + 1, jid.indexOf('/'));
 
-        const xEl = pres.querySelectorAll('x')[0];
+        const xEl = pres.querySelector('x');
```

Comment:
so this just gets the first result?

Reply:
Yeah, it just gets the first result.

Reply:
Well one difference is what's returned. If nothing is found, querySelector will return null/falsy. querySelectorAll will return an empty array.